DeAceroDLabs/marketplace-frontend
# fix: [DMP-742] fix stepper line on signup.

## What is done

[[DMP-742] fix stepper line on signup.](https://deacero.atlassian.net/browse/DMP-742)

## How is done

- Added z-index: 1;
- Added validation to check variant type.

## How to test

1. Run the project via `npm start`.
2. Go to `registrarse`.
3. you should see the line displayed correctly as this:
![image](https://user-images.githubusercontent.com/81311646/208998983-d9aa5f14-48f0-4489-831d-02d9d0aa2afa.png)

### Evidence:

before:
![image](https://user-images.githubusercontent.com/81311646/209000677-56764f95-44d0-4923-ab40-8ce994b84707.png)

after:

![image](https://user-images.githubusercontent.com/81311646/208998983-d9aa5f14-48f0-4489-831d-02d9d0aa2afa.png)
